scripts/novaseq/stage/checkfornewrun.bash

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -u
+
+shopt -s nullglob
+
+########
+# VARS #
+########
+
+IN_DIR=${1?'please provide the runs dir'}
+DEMUXES_DIR=${2?'please provide the demuxes dir'}
+SCRIPT_DIR=$(dirname $(readlink -nm $0))
+#EMAIL=clinical-demux@scilifelab.se
+EMAIL=your.name@scilifelab.se
```

Comment:
Is this right? 🙉

Reply:
yes, it's stage ready scripts (notice in which dir this script lives), for development and testing, and soon when we move to python, for throwing away. Just fill in your own name when you work on it and save the clinical-demux email address from getting spammed.

Reply:
I assume you mean lines 14 and 15 btw, that wasn't very clear.